constants/constants.go

```diff
@@ -22,8 +22,11 @@ const (
 	Cloudline = "Cloudline"
 	// Quanta is the contant to identify Quanta hardware
 	Quanta = "Quanta"
-	// Quanta is the contant to identify Intel hardware
+	// Intel is the contant to identify Intel hardware
 	Intel = "Intel"
+	// Equinix, formerly Packet
```

Comment:
what are these identifying? does Packet and Equinix have hardware like Intel and or Quanta do?

Reply:
maybe this is related to a customer openBMC build Equinix/Packet are running with redfish in it? If this is the case we should make that clear here. Without it I'm left to assume this is a server that built by Equinix/Packet like intel or dell.

Reply:
In this case the hardware was manufactured with the vendor attribute set to `Packet`.